public/data/SyedSohan.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Syed Sohan",
+  "bio": "Project Executive - IT | Tech Enthusiast",
+  "avatar": "https://github.com/syedsohan.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/syedsohan",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/syedsohan27",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/syedsohan",
+      "icon": "linkedin"
+    }
+  ]
+}
```

Still missing an endline.
```suggestion
}

```

---

Added.